packages/number-field/src/number-field.css

```diff
@@ -13,6 +13,9 @@ governing permissions and limitations under the License.
 @import url('./spectrum-number-field.css');
 @import url('./number-field-overrides.css');
 
+:host {
+    inline-size: unset;
```

Comment:
Sp-text-field sets an `inline-size` on the host, which causes side effects for sp-slider that uses number-field. This unsets the inline size and resolves the issue PSWeb called out on sp-slider [editable]. 

Fun fact: number-field extends textFieldBase, so moving forward, any changes to the text field will need to cross-check the number field heavily.